hs_core/hydroshare/resource.py

```diff
@@ -438,8 +444,8 @@ def create_empty_resource(pk, user, action='version'):
         user: the user who requests to create a new version for the resource or copy the resource.
         action: "version" or "copy" with default action being "version"
     Returns:
-        the empty new resource that is created as an initial new version or copy for the original
-        resource which is then further populated with metadata and content in a subsequent step
+        the empty new resource that is created as an initial new version for the original resource
+        which is then further populated with metadata and content in a subsequent step
```

Comment:
@alvacouch This function is used by both creating new version of a resource and creating a new copy of a resource, so the original comment should be kept or made better but not remove ```copy``` since it is important to note this function is used by both functionalities.